### PR TITLE
update cli 'account withdraw' interface

### LINKED ISSUE #2880: Fix help message to "account withdraw" command
> account withdraw
the following arguments are required: amount, destination, currency

Should be: the following arguments are required: destination amount currency. For eg. account withdraw 0x74446B4FB2Cca2fd8A475Bc40606b77d8d7659C4 3 GNT

Also, information about locked withdraws on the testnet should be shown earlier for the user, not after he types all necessary commands. Maybe just add comment in account “withdrawals are not available for the testnet”

### PR DESCRIPTION
**Pull request by etam**
Fixes #2880

### CODE REVIEW

**Review by jiivan — APPROVED** [review]
lgtm

File: golem/interface/client/account.py [hunk 1]
```diff
@@ -102,12 +102,13 @@ def unlock(self) -> str:  # pylint: disable=no-self-use
...
-        arguments=(amount_arg, address_arg, currency_arg),
-        help="Withdraw GNT/ETH")
+        arguments=(address_arg, amount_arg, currency_arg),
+        help=("Withdraw GNT/ETH\n"
+              "(withdrawals are not available for the testnet)"))
     def withdraw(  # pylint: disable=no-self-use
             self,
-            amount,
             destination,
+            amount,
```
Comment (Krigpl) [inline golem/interface/client/account.py:111]: Why the reorder? All other `withdraw` functions have `amount` first

Comment (Krigpl) [reply golem/interface/client/account.py:111]: I found the answer so nevermind

Comment (etam) [reply golem/interface/client/account.py]: hat's what #2880 is about. And this is only a CLI command, that's why I'm not reordering further.

**Pull request merged**